README.md

```diff
@@ -334,6 +343,7 @@ grafana_dashboard { 'example_dashboard':
   grafana_url       => 'http://localhost:3000',
   grafana_user      => 'admin',
   grafana_password  => '5ecretPassw0rd',
+  grafana_api_path  => '/api'
```

Comment:
'/api' is the default value for grafana_api_path. I would add in a sub-path to show the use of the parameter, something like '/subpath/api' or '/grafana/api'